module/checker.go

```diff
@@ -102,8 +102,6 @@ func (m *Module) CheckFieldRules(typ FieldType, rules *validate.FieldRules) {
 	case *validate.FieldRules_Enum:
 		m.MustType(typ, pgs.EnumT, pgs.UnknownWKT)
 		m.CheckEnum(typ, r.Enum)
-	case *validate.FieldRules_Message:
```

Comment:
I think a big thing we're missing in this PR is checking whether the message validate is compatible. 

For example, I don't believe it would be compatible with:
- AnyRules
- DurationRules
- TimestampRules
- EnumRules
- MapRules

I also bet it will take more to implement this for RepeatedRules - should check on this.

Reply:
And we also want to make sure that MessageRules is only applied to messages. This deletion removed that.